schema/entity-record.json

```
@@ -12,21 +12,117 @@
       "type": "boolean"
     },
     "entityType": {
+      "entityType": "object",
       "title": "Type",
-      "description": "From the entityType codelist. What kind of entity is this? The 'registeredEntity' code covers any legal entity created through an act of official registration, usually resulting in an identifier being assigned to the entity. The ‘legalEntity’ code covers other bodies with distinct legal personality (international institutions, statutory corporations etc.). The 'arrangement' code covers artificial entities, described in the data model for the purpose of associating one or more natural or legal persons together in an ownership or control relationship, but without implying that the parties to this arrangement have any other form of collective legal identity.",
-      "type": "string",
-      "enum": [
-        "registeredEntity",
-        "legalEntity",
-        "arrangement",
-        "anonymousEntity",
-        "unknownEntity",
-        "state",
-        "stateBody"
+      "description": "The type of entity described in the statement.",
```

```suggestion
      "description": "The form of the entity described in the Statement.",
```
I'm suggesting edits for these field descriptions on your PR, instead of on my schema description editing branch.